frontend/src/components/NavBar.tsx

```diff
@@ -72,7 +72,7 @@ export function NavBar() {
             </a>
           </li>
           <li>
-            <NavLink className={styles.navLink} to={"/apply"}>
+            <NavLink className={styles.navLink} to={"/prescreening"}>
               <Button onClick={null}>Apply</Button>
```

Comment:
There are also a few buttons on the home page [here ](https://github.com/TritonSE/CCIDC-Application-Overhaul/blob/main/frontend/src/pages/Candidates.tsx#L59) and [here ](https://github.com/TritonSE/CCIDC-Application-Overhaul/blob/main/frontend/src/pages/Candidates.tsx#L107) to update.

You can either use the useNavigate method you did in the `handleSubmit` or the `window.location.href` way used in the [Congradulations popup](https://github.com/TritonSE/CCIDC-Application-Overhaul/blob/main/frontend/src/components/CongratulationsPopupWindow.tsx#L22)